locale/de/base.cfg

```diff
@@ -37,39 +37,42 @@ ltn-position-any-artillery-wagon=Kodierte Reihung aller Artilleriekanonen
 ltn-position=Kodierte Reihung von __1__
 ltn-depot=Haltestelle ist Depot
 ltn-depot-priority=Depotpriorität
-ltn-network-id=Kodierte Netzwerk ID
-ltn-min-train-length=minimale Zuglänge
-ltn-max-train-length=maximale Zuglänge
-ltn-max-trains=maximale Zuganzahl
+ltn-network-id=Kodierte Netzwerk-ID
+ltn-min-train-length=Minimale Zuglänge
+ltn-max-train-length=Maximale Zuglänge
+ltn-max-trains=Maximale Zuganzahl
 ltn-requester-threshold=Mindestmenge für Anforderung
-ltn-requester-stack-threshold=Minimum Stapel für Anforderung
+ltn-requester-stack-threshold=Mindestmenge der Stapel für Anforderungen
 ltn-requester-priority=Anforderungspriorität
 ltn-provider-threshold=Mindestmenge für Angebot
-ltn-provider-stack-threshold=Minimum Stapel für Angebot
+ltn-provider-stack-threshold=Mindestmenge der Stapel für Angebote
 ltn-provider-priority=Anbieterpriorität
-ltn-locked-slots=gesperrte Stapel pro Waggon
-ltn-disable-warnings=Warnungen deaktivieren
+ltn-locked-slots=Gesperrte Waggonplätze

```

Comment:
unpräzise ob gesperrte Plätze pro Zug oder Waggon

In ltn-depot-reset-filters benutzt du "Stapelbegrenzungen in Güterwaggons" warum stört dich hier meine Übersetzung?